edxapp/Dockerfile

```diff
@@ -23,15 +23,19 @@ VOLUME /openedx/data
 WORKDIR /openedx/edx-platform
 
 ## Checkout edx-platform code
-RUN git clone https://github.com/edx/edx-platform.git --branch open-release/ginkgo.master --depth 1 .
+ARG EDX_PLATFORM_REPOSITORY=https://github.com/edx/edx-platform.git
```

Comment:
This is nice because it also makes it easier to use a forked version of edx-platform.